### PR TITLE
WRP-567: Deprecate `value` prop to be replaced by `defaultValue` in `InputPopupBase`

### PR DESCRIPTION
**Pull request by MikyungKim**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`value` prop in `InputPopupBase` is used for the initial value but the name of the prop is somewhat confusing.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Deprecate `value` prop to be replaced by the newly added `defaultValue`. 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
The description of `value` in `Input` component is incorrect so I fixed it.
I've added documentation of `defaultValue` in `Input` component.

### Links
[//]: # (Related issues, references)
WRP-567

### Comments
Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,15 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ## [unreleased]
 
+### Deprecated
+
+- `sandstone/Input.InputPopupBase` prop `value`, to be removed in 3.0.0. Use `sandstone/Input.InputPopupBase` prop `defaultValue` instead.
```
Comment (0x64) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Input.InputPopupBase` prop `value`, to be removed in 3.0.0. Use `defaultValue` instead.
```

Comment (0x64) [reply CHANGELOG.md:9]: I prefer a simpler one if clear enough. ;)

File: Input/Input.js [hunk 2]
```diff
@@ -566,7 +595,7 @@ const InputBase = kind({
 					type={type}
 					size={size}
 					disabled={disabled}
-					value={value}
+					defaultValue={value}
```
Comment (0x64) [inline Input/Input.js:598]: Let's remove this line to prevent future misunderstanding.

**Review by 0x64 — CHANGES_REQUESTED** [review]

File: Input/Input.js (hunk 2 above)
Comment (0x64) [reply Input/Input.js:598]: Closing after discussion: we need this.

File: CHANGELOG.md (hunk 1 above)
Comment (MikyungKim) [reply CHANGELOG.md]: ✔️

**Review by 0x64 — APPROVED** [review]
💡

**Review by seunghoh — APPROVED** [review]

**Pull request merged**